src/components/SpruceForm/errors.ts

```diff
@@ -28,7 +28,7 @@ export const transformErrors = (errors: AjvError[]) =>
         case "minLength":
           return {
             ...error,
-            message: "",
+            message: "invisible",
```

Comment:
[nit] Perhaps we could use a string enum to reduce the magic string-ness of this admittedly hacky solution 😄